src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -0,0 +1,214 @@
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 58,
+which is in beta as of March 16. This list is subject to change at any time.
+
+
+## Remove RTCRtcpMuxPolicy of "negotiate"
+
+The `rtcpMuxPolicy` is used by Chrome to specify its preferred policy regarding
+use of RTP/RTCP multiplexing. In Chrome 57, we changed the default
+`rtcpMuxPolicy` to "require" and deprecated "negotiate" for following reasons:
+
+* Non-muxed RTCP uses extra network resources.
+* Removing "negotiate" will make the API surface simpler, since an 
+  "RtpSender"/"RtpReceiver" will then only ever have a single transport.
+
+In Chrome 58, "negotiate" is removed. We believe this is a non-breaking change
+since the user will get a deprecation message and the `webkitRTCPeerConnection`
+can still be created successfully.
+
+[Intent to Deprecate](https://groups.google.com/a/chromium.org/d/topic/blink-dev/OP2SGSWF5lo/discussion) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=685727)
+
+
+## Mouse on Android stops firing TouchEvents
+
+Until Chrome 55, Android low-level mouse events in Chrome primarily followed an
+event path designed for touch interactions. For example, mouse drag motion while
+a mouse button is pressed generates `MotionEvents` delivered through
+`View.onTouchEvent`.
+
+However, since touch events cannot support hover, hovering mousemoves followed a
+separate path. The design had many side-effects including mouse interactions
+firing `TouchEvents`, all mouse buttons appearing as *left* mouse buttons, and
+`MouseEvents` being suppressed by `TouchEvents`.
+
+Starting with Chrome 58, a mouse on Android M or later will:
+
+* No longer fire `TouchEvents`.
+* Fire a consistent sequence of `MouseEvents` with appropriate buttons and
+  other properties. 
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/cNaFvMaYtNA/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5642080642662400) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=468806)
+
+
+## Remove case-insensitive matching for usemap attribute
+
+The `usemap` attribute was formerly defined as caseless. Unfortunately
+implementing this was complicated enough that no browsers implemented it
+correctly. Research suggested that such a complicated algorithm is unnecessary,
+and even ASCII case-insensitive matching is unnecessary.
+
+Consequently, the specification was updated so that case-sensitive matching is
+applied. The old behavior was deprecated in Chrome 57, and is now removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/8pHdFzN0YQc/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5760965337415680) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=659464)
+
+
+## Remove EME from unsecure contexts
```

Comment:
"non-secure" is what we usually try to say. "insecure" has strange psychological connotations about the way an origin feels about its own self-worth. "unsecure" sounds weird. :)